optimum/neuron/generation/utils.py

```diff
@@ -14,8 +14,840 @@
 # limitations under the License.
 """Utilities for generation with Neuron."""
 
+import copy
+import inspect
+import warnings
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.distributed as dist
+import torch_xla.core.xla_model as xm
```

Comment:
This causes an import error.
Can you do:

```
from ..utils import is_torch_xla_available
if is_torch_xla_available():
    import torch_xla.core.xla_model
```
Basically we want to be able to import and test code on regular machines when we do not need all of this.

Reply:
Got it, is the optimum neuron library designed for both CPU and Trainium?

Reply:
Only Trainium but it is more to be able to run non-trainum dependent tests on regular machines. Without this the test will fail if we dont have `torch_xla` installed even though we want to test something unrelated.